install.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+
+node-pre-gyp install 2> /dev/null || \
+	([ -d "libusb" ] || git clone --depth=1 https://github.com/kevinmehall/libusb.git) && \
```

Comment:
it doesn't work. try it. it just returns without doing anything. I did some research and everyone else is cloning